db/models/evaluationItem.ts

```diff
@@ -1,14 +1,15 @@
 /* eslint-disable no-use-before-define */
 import {
   Model,
-  InferAttributes,
-  InferCreationAttributes,
   DataTypes,
-  CreationOptional,
-  type NonAttribute,
   Association,
+  type InferAttributes,
+  type InferCreationAttributes,
+  type CreationOptional,
+  type NonAttribute,
+  type BelongsToManyAddAssociationMixin,
 } from "sequelize";
-import type { Milestone } from ".";
+import type { Milestone, User } from ".";
 
 type levels = "Very High" | "High" | "Moderate" | "Low" | "Very Low";
```

Comment:
rename to whatever the source document is so that it's easier to know that these are like the POAM kind of levels